docs/markdown/Dependencies.md

```diff
@@ -148,6 +148,25 @@ environment variables.
 You can set the argument `threading` to `single` to use boost libraries that
 have been compiled for single-threaded use instead.
 
+## Dub
+
+Use `method` to find dependencies with Dub. Just create a dependency as you would normally, but add `dub` as the dependency method.
+```meson
+urld_dep = dependency('urld', method: 'dub')
```

Comment:
Hmm, I don't think that this will cover all cases well enough.
Dub allows projects to build subconfigurations, and that would have to be reflected in the dependency declaration somehow (for example, for a database connector you might only want its Postgresql variant, or you want to compile some other project without Vibe.d dependency). So additional to `method` we also might need a `config` specifier.
As soon as subconfigurations are suppoorted, the name of the dependency can't be fetched anymore via pkg-config though, because the pkg-config stuff often has a different name (e.g. "ddbc-posgresql" for a library compiled for a specific subconfiguration).
So, I think we'll need a solution for that. As long as we can add a feature like that later easily without breakage, merging the PR is fine though (we'll just need to keep in mind how subconfigurations are implemented in a way that isn't messy).

Reply:
Sounds good to me.